app/sidekiq/decision_review/failure_notification_email_job.rb

```diff
@@ -0,0 +1,140 @@
+    end
+
+    # Fetches SavedClaim records for DecisionReview that have an error status for the form or any evidence attachments
+    def errored_saved_claims
+      @errored_saved_claims ||= ::SavedClaim.where(type: SAVED_CLAIM_MODEL_TYPES)
+                                            .where(delete_date: nil)
+                                            .where('metadata LIKE ?', '%error%')
+                                            .order(id: :asc)
+    end
+
+    def submissions
+      @submissions ||= begin
+        guids = errored_saved_claims.select { |sc| JSON.parse(sc.metadata)['status'] == ERROR_STATUS }.pluck(:guid)
+        ::AppealSubmission.where(submitted_appeal_uuid: guids).failure_not_sent
+      end
+    end
+
+    def submission_uploads
+      @submission_uploads ||= begin
+        uploads = errored_saved_claims.map { |sc| JSON.parse(sc.metadata)['uploads'] }
+        ids = uploads.flatten.select { |upload| upload&.fetch('status') == ERROR_STATUS }.pluck('id')
+
+        ::AppealSubmissionUpload.where(lighthouse_upload_id: ids).failure_not_sent
+      end
+    end
+
+    def send_email_with_vanotify(submission, filename, created_at)
+      email_address = submission.current_email
+      personalisation = {
+        first_name: submission.get_mpi_profile.given_names[0],
+        filename:,
+        date_submitted: created_at.strftime('%B %d, %Y')
+      }
+
+      vanotify_service.send_email({ email_address:, template_id: TEMPLATE_IDS[submission.type_of_appeal],
+                                    personalisation: })
+    end
+
+    def send_form_emails
+      StatsD.increment("#{STATSD_KEY_PREFIX}.form.processing_records", submissions.size)
+
+      submissions.each do |submission|
+        send_email_with_vanotify(submission, nil, submission.created_at)
+        submission.update(failure_notification_sent_at: DateTime.now)
+
+        record_form_email_send_successful(submission)
+      rescue => e
+        record_form_email_send_failure(submission, e)
+      end
+    end
+
+    def send_evidence_emails
+      StatsD.increment("#{STATSD_KEY_PREFIX}.evidence.processing_records", submission_uploads.size)
+
+      submission_uploads.each do |upload|
+        send_email_with_vanotify(upload.appeal_submission, upload.masked_attachment_filename, upload.created_at)
+        upload.update(failure_notification_sent_at: DateTime.now)
+
+        record_evidence_email_send_successful(upload)
+      rescue => e
+        record_evidence_email_send_failure(upload, e)
+      end
+    end
+
+    def record_form_email_send_successful(submission)
+      metadata = { submitted_appeal_uuid: submission.submitted_appeal_uuid, form_type: submission.type_of_appeal }
+      Rails.logger.info('DecisionReview::FailureNotificationEmailJob form email queued', metadata)
+      StatsD.increment("#{STATSD_KEY_PREFIX}.form.email_queued", tags: ["form_type:#{submission.type_of_appeal}"])
```

Comment:
When do we set the `delete_date` on the SavedClaim? Also, where does `failure_notification_sent_at` get set on the AppealSubmission?

Reply:
`delete_date` is set in other Sidekiq jobs (the SavedClaim status updater jobs [here](https://github.com/department-of-veterans-affairs/vets-api/tree/master/app/sidekiq/decision_review)). 

`failure_notification_sent_at` is set on line 83.